docs/boot-options.rst

```diff
@@ -658,13 +658,6 @@ will be used by default, if found.
 
 See the |anacondalogging|_ for more info on setting up logging via virtio.
 
-.. inst.zram:
-
-inst.zram
-^^^^^^^^^
-
-Forces/disables (on/off) usage of zRAM swap for the installation process.
-
```

Comment:
Please move this to the `Removed options` section.

Reply:
Good point! Updated.